crates/ubrn_bindgen/src/bindings/react_native/mod.rs

```diff
@@ -378,6 +387,37 @@ impl FfiType {
     fn is_void(&self) -> bool {
         matches!(self, Self::VoidPointer)
     }
+
+    fn cpp_namespace(&self, ci: &ComponentInterface) -> String {
```

Comment:
This describes which  namespace the struct declaration is in.

The namespace where the `Bridging` class is in `bridging_namespace`.